pkg/disk/disk-unix.go

```diff
@@ -29,3 +29,15 @@ func IsRootDisk(diskPath string) (bool, error) {
 	}
 	return rootDisk, nil
 }
+
+// DiskDeviceID - returns device-id of the mounted disk.
+func DiskDeviceID(diskPath string) (uint64, error) {
```

Comment:
also the function should be `disk.DeviceID`  Disk prefix for the function is not needed since this is a disk package.